06_modules_lab/01.py

```diff
@@ -1,4 +1,5 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+from sys import argv
+
+(script_name, lines) = argv

```

Comment:
Thanks! I was actually wondering about the _ usage in python scripts... That's very helpful :+1: